R/animint.R

```diff
@@ -94,7 +94,9 @@ parsePlot <- function(meta){
       ##    theme.pars$panel.grid then from theme.pars$line
       for(i in 1:length(pars)) {
         if(is.null(pars[[i]])) pars[[i]] <- 
-          if(!is.null(theme.pars$panel.grid[[i]])) {
+          if("element_blank" %in% class(theme.pars$panel.grid)) {
+            theme.pars$line[[i]]
+          } else if( !is.null(theme.pars$panel.grid[[i]]) ) {
```

Comment:
What I want to do is say "if the parameter is NULL then look for it in `theme.pars$panel.grid`, if it's NULL there then look for it in `theme.pars$NULL`.

There's an error, however, if `$panel.grid` is an empty list - i.e. if it's created by `element_blank()`.
```
x <- list()
x[[1]]
Error in x[[1]] : subscript out of bounds
```

So I believe I first need to check whether it's specified as `element_blank()` or not.  But is there a better way to do this?